### PR TITLE
[SENT 16/10/17] Digest

### PR DESCRIPTION
**Pull request by nvgrw**
Digest for 2017-10-16.

[Draft 1 PDF](https://github.com/icdocsoc/emails/files/1385613/2017-10-16.D1.pdf)

### CODE REVIEW

File: markdown/2017-10-16.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+- **Where:** Metric
+- **Link:** [Facebook](https://www.facebook.com/events/857222977758027/)
+
+It's time for DoCSoc's first bar night of the year: The Welcome Bar Night!
+
+Come along for free drinks and to say hi to our new freshers or catch up with
+friends - it's sure to be a night you don't want to miss out on.
+
+Drinks tokens will be given upon entry for FREE to all DoCSoc members who
+acquired membership before 18.00 on the day of the event. Strictly 18+ only.
```
Comment (jamesprinc3) [inline markdown/2017-10-16.md]: worth reminding people that DoC, EIE, JMC are automatically members?

File: markdown/2017-10-16.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+Every year students vote for at least one Academic Year Representative who
+will receive and collate feedback from their cohort and will pass this
+information to the Departmental Representatives or staff as appropriate.
+Academic Representatives have made long-lasting improvements in many areas
+such as the content of curricula, feedback quality and timeliness, exam
+timetables and formats, online learning resources, and the physical
+facilities available.
+ 
+This year the Union has also newly launched a Wellbeing Representation
```
Comment (jamesprinc3) [inline markdown/2017-10-16.md]: delet "newly"

File: markdown/2017-10-16.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+Network, which aims to tackle issues such as stress and pressure, mental
```
Comment (jamesprinc3) [inline markdown/2017-10-16.md]: can we hyperlink "Wellbeing Representation Network" to some union page?

File: markdown/2017-10-16.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+# ICSM Coding Looking for Volunteer Teaching Assistants
+
+ICSM Coding Society (brand new this year) aims to provide easy access to
+beginner-level coding tutorials for students with no previous coding
+experience. More info here:
+<https://www.imperialcollegeunion.org/activities/a-to-z/coding-medics> and
+<https://www.facebook.com/ICSMCoding/>
+
+We are running a series of Python coding workshops in November aimed at
```
Comment (jamesprinc3) [inline markdown/2017-10-16.md]: Can you quote all of this to make it clear that this wasn't written by docsoc

File: markdown/2017-10-16.md (hunk 2 above)
Comment (nvgrw) [reply markdown/2017-10-16.md]: y delet

File: markdown/2017-10-16.md (hunk 3 above)
Comment (nvgrw) [reply markdown/2017-10-16.md]: Could do, but the main elections page has deets on this already and that's linked a bit later. Would that not be redundant?

File: markdown/2017-10-16.md (hunk 1 above)
Comment (nvgrw) [reply markdown/2017-10-16.md]: Yes good idea

Comment (nvgrw) [reply markdown/2017-10-16.md]: Computing, JMC, EIE = DoC tho 😛

File: markdown/2017-10-16.md (hunk 4 above)
Comment (nvgrw) [reply markdown/2017-10-16.md]: I think I want to shorten it too, it's a bit too long. I'll make it sound like it's not coming from us. I don't think the current preprocessor and stylesheet supports blockquotes too well

File: markdown/2017-10-16.md (hunk 2 above)
Comment (levex) [reply markdown/2017-10-16.md]: wat iz newly

File: markdown/2017-10-16.md (hunk 1 above)
Comment (mauriceyap) [reply markdown/2017-10-16.md]: EIE aren't technically DoC - the course is run solely by EEE Dept

File: markdown/2017-10-16.md (hunk 2 above)
Comment (mauriceyap) [reply markdown/2017-10-16.md]: This year the Union has also _launched a new_ Wellbeing...

File: markdown/2017-10-16.md (hunk 3 above)
Comment (mauriceyap) [reply markdown/2017-10-16.md]: I think link it anyway

**Pull request merged**